### PR TITLE
clarified the use of shared port in GWMS and the details about how to change HTCondor configuration

### PR DESCRIPTION
**Pull request by mambelli**

### CODE REVIEW

File: docs/release/3.4/release-3-4-28.md [hunk 1]
```diff
@@ -62,16 +62,18 @@ Detailed changes are below. All of the documentation can be found [here](/index.
...
+1. GlideinWMS >= 3.4.5 can use shared port. If you do so, it requires only port 9618.
    To ease the transition to shared port, the User Collector secondary collectors and CCBs support both shared and
    separate, individual ports.
-   To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in
-   `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:
 
+   -  The HTCondor configuration should be changed as suggested in the new HTCondor config files included in the RPMs, to allow the support of shared port: check the `.rpmnew` files or apply the same changes if you configure your daemons manually.
+   -  Port 9818 should be open also on standalone schedds (that earlier used only 9615).
+   -  To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:
+      ```
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-28.md]: Triple backticks don't work within lists. Instead, you need to indent code blocks by 4 spaces per list level (https://opensciencegrid.org/technology/documentation/style-guide/#lists)

```suggestion

```

File: docs/release/3.4/release-3-4-28.md [hunk 2]
```diff
@@ -62,16 +62,18 @@ Detailed changes are below. All of the documentation can be found [here](/index.
...
         <collector DN="/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=gwms-frontend.domain" group="default" node="gwms-frontend.domain:9618?sock=collector0-40" secondary="True"/>
-
-   Replacing `gwms-frontend-domain` with the hostname of your GlideinWMS frontend.
-   See the [GlideinWMS documentation](https://glideinwms.fnal.gov/doc.prd/components/condor.html#collectors ) for details. 
+      ```
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-28.md]: Remove backticks

```suggestion

```

**Comment by brianhlin** [comment]
@timtheisen thoughts?

File: docs/release/3.4/release-3-4-28.md [hunk 3]
```diff
@@ -62,16 +62,18 @@ Detailed changes are below. All of the documentation can be found [here](/index.
...
 1. Upgrades from <= 3.4.0 may require merging `/etc/condor/config.d/*.rpmnew` files and a restart of HTCondor.
 
-1. GlideinWMS >= 3.4.5 uses shared port, requiring only port 9618.
+1. GlideinWMS >= 3.4.5 can use shared port. If you do so, it requires only port 9618.
    To ease the transition to shared port, the User Collector secondary collectors and CCBs support both shared and
    separate, individual ports.
-   To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in
-   `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:
 
+   -  The HTCondor configuration should be changed as suggested in the new HTCondor config files included in the RPMs, to allow the support of shared port: check the `.rpmnew` files or apply the same changes if you configure your daemons manually.
```
[the remark this answers is not in the document]
Comment (mambelli) [reply docs/release/3.4/release-3-4-28.md]: Line 63 is incomplete. There were changes to the HTCondor config also in 3.4.4. And I found out that some people have components like the user collector on a separate machine and do not use GWMS RPMs there. That's the reason for the "apply the same changes sentence". Should I do these changes to on line 63?

[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/release/3.4/release-3-4-28.md]: Yeah, I'd say something like the following:

```
1. Upgrades from <= 3.4.0 may require merging `/etc/condor/config.d/*.rpmnew` files, or applying those same changes if you configure your daemons manually, and a restart of HTCondor.
```

[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/release/3.4/release-3-4-28.md]: @mambelli see my comment above

[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/release/3.4/release-3-4-28.md]: @mambelli note that your changes still haven't been merged

[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/release/3.4/release-3-4-28.md]: @mambelli please make the changes I requested above

**Pull request merged**